bedrock/base/templates/includes/protocol/navigation/nav-cta.html

```diff
@@ -8,7 +8,7 @@
   <div class="c-navigation-shoulder">
     {% if not custom_nav_cta %}
       {% if not hide_nav_download_button %}
-        {{ download_firefox_thanks(alt_copy=ftl('download-button-download-firefox'), dom_id='protocol-nav-download-firefox', button_class='mzp-t-secondary mzp-t-md', download_location='nav') }}
+        {{ download_firefox_thanks(alt_copy=ftl('download-button-download-firefox'), dom_id='protocol-nav-download-firefox', button_class='mzp-t-secondary mzp-t-md js-glea-delay-click', download_location='nav') }}
```

Comment:
typo on the glean class
```suggestion
        {{ download_firefox_thanks(alt_copy=ftl('download-button-download-firefox'), dom_id='protocol-nav-download-firefox', button_class='mzp-t-secondary mzp-t-md js-glean-delay-click', download_location='nav') }}
```